alica_standard_library/include/Conditions.h

```diff
@@ -21,6 +22,7 @@ BOOST_DLL_ALIAS(alica_standard_library::AnyChildFailure, AnyChildFailure)
 BOOST_DLL_ALIAS(alica_standard_library::AllChildFailure, AllChildFailure)
 BOOST_DLL_ALIAS(alica_standard_library::AlwaysTrueCondition, AlwaysTrueCondition)
 BOOST_DLL_ALIAS(alica_standard_library::AlwaysFalseCondition, AlwaysFalseCondition)
+BOOST_DLL_ALIAS(alica_standard_library::Equals, Equals)
```

Comment:
Maybe CompareString is a better name.  Alternatively, we make it more general to support more different types:  Maybe we can just compare both elements as variants?